README.md

```diff
@@ -11,28 +11,25 @@ Inschrijvingssysteem voor Sint Gerardus
 * Beheren van verschillende types gebruikers:
   * Administrators
   * Zorggebruikers
-  * Voogde
+  * Voogden
   
 ### Evenementenbeheer
 
 * Beheren van evenementen
-  * Zorggebruikers kunnen zich inschrijven voor evenementen
+  * Voogen/ouders van zorggebruikers kunnen hun kind inschrijven voor evenementen
   * Verschillende leeftijdsgroepen toekennen aan evenementen
   * Evenementen kunnen over één of meerdere dagen plaatsvinden
 
-
-
 ## Setup (English only)
 
 ### Code
 
 1. Architecture
 
-The application is written in C#, using .NET Core 2.2 and Angular 7. It's is generated using a template provided by Visual Studio 2017.
+The application is written in C#, using .NET Core 2.2 and Angular 8. It's is generated using a template provided by Visual Studio 2017.
```

Comment:
![afbeelding](https://user-images.githubusercontent.com/41837971/74463972-ffade680-4e92-11ea-9854-2536ac4c3f87.png)

We still use Angular 7.